### PR TITLE
fix #77481 teknikhukuk.com

### LINKED ISSUE #77481: www.teknikhukuk.com
### Issue URL (Annoyance)
 [https://www.teknikhukuk.com/internet-bankaciligi-kullananlar-dikkat-yargitaydan-onemli-karar/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.teknikhukuk.com%2Finternet-bankaciligi-kullananlar-dikkat-yargitaydan-onemli-karar%2F)
### Comment
Username: @RadicalNoise
> `g1-slideup-newsletter`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#77481

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -104,6 +104,8 @@ academictimes.com###cover
 stuff.co.nz##body > div.t5.t6
 economist.com###piano-ribbon
 fromsmash.com###welcome-modal
+teknikhukuk.com#$#.g1-popup { display: none; !important }
+teknikhukuk.com#$#body { overflow: visible; !important; }
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/popups.txt:108]: Unnecessary semicolons before the exclamation points, and please change body to html

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]
Please also block the newsletter in the sidebar

**Comment by wout-codes** [comment]
> Please also block the newsletter in the sidebar

I am not sure how. It may block the whole sidebar.

**Comment by Sergey-Lyapin** [comment]
> > Please also block the newsletter in the sidebar
> 
> I am not sure how. It may block the whole sidebar.

##.g1-newsletter

**Pull request merged**